src/jsMain/kotlin/dev/bitspittle/firebase/auth/Bindings.ActionInfo.kt

```diff
@@ -0,0 +1,27 @@
+package dev.bitspittle.firebase.auth
+
+import dev.bitspittle.firebase.externals.auth.ActionCodeSettings
+import dev.bitspittle.firebase.util.jsonWithoutNullValues
+
+class ActionCodeSettings(
+    android: Android,
+    dynamicLinkDomain: String?,
+    override val handleCodeInApp: Boolean,
+    iOS: Ios,
+    override val url: String
+): ActionCodeSettings {
+    class Android(val installApp: Boolean?, val minimumVersion: String?, val packageName: String)
```

Comment:
I actually originally named these `AndroidSettings` and `IosSettings` but renamed it to keep consistent with the JS naming.
I am fine with naming you proposed 👍

Reply:
Yeah, I can appreciate the way you thought here too. 

I really don't like how JS/TS libraries just use arbitrary JSON objects as config values, it results in a sloppy mess in so many APIs :) But this is ancient history, nothing we can do about it. I don't really know what name they would have chosen if they specified it.

I tried looking for inspiration from elsewhere in the library, but ugh look at this: https://firebase.google.com/docs/reference/js/app.md#initializeapp_3

They use "options", "config", and "settings" all in a single signature!?!?! (Bad Google, no cookie for you)

And here they use config: https://firebase.google.com/docs/reference/js/auth.config
But here they use settings: https://firebase.google.com/docs/reference/js/auth.authsettings
And here they use options: https://firebase.google.com/docs/reference/js/auth.oauthcredentialoptions

There's no clear guidance for which word means what. I think "Config" and "Settings" both work here, so I'll leave the final call to you.

Reply:
Yeah, on Android they don't even expose the constructor. Just a builder and convenient Kotlin scope function

![image](https://user-images.githubusercontent.com/11587878/230976909-9008abed-2d13-4e92-a139-32f39ecf6aa6.png)

so sadly we can't even take inspiration in typed languages (the class file is obfuscated so even that doesn't work).
Personally I think that my original approach of just naming it `Android` nested class is the best because we wouldn't have these debates. It would just be `ActionCodeSettings.Android`. But let's leave it as it is. It can always be renamed if you settle on some convention.